envi/archs/msp430/disasm.py

```diff
@@ -489,7 +488,10 @@ def getOperValue(self, op, emu=None):
         if emu==None: 
             return
 
-        return emu.getRegister(self.val)
+        val = emu.getRegister(self.val)
```

Comment:
if you're checking for REG_PC, this should go above the emulator check above, since we always know PC.